backend/src/auth/auth.controller.ts

```diff
@@ -48,4 +61,47 @@ export class AuthController {
   signin(@Body() dto: AuthDto) {
     return this.authService.signin(dto);
   }
+
+  @UseGuards(FortyTwoGuard)
+  @Get('42')
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  async fortyTwoAuth() {}
```

Comment:
Why is this needed?
Is it the callback URL?

Reply:
No, it's empty function to call the 42 strategy who redirect to the callback.
It's ugly but everyone do this like that...